### DISCUSSION TITLE
config debug isn't working

### DISCUSSION THREAD

**Discussion by windowsyl** [kovidgoyal/kitty#5088] [Q&A] [answered]
I installed kitty with brew and when I press cmd +  opt + , (on mac) i get an error: 
Key action failed
debug_config
invalid literal for int() with base 10: ''
Press Enter to close

I've commented out my config, completely deleted it, uninstalled and reinstalled kitty with homebrew, deleted any cache files i could find with brew or with AppCleaner. I've been doing this next to a friend who also uses kitty and pretty identical systems and his config debug works fine. Is there something obvious we're missing? I can't think of anything else to try deleting and reinstalling to get this to work. Any help or ideas would be appreciated

**Comment by kovidgoyal** [comment, marked as answer]
See #5051
